src/runtime_src/core/edge/sw_em/generic_pcie_hal2/shim.cxx

```diff
@@ -80,7 +80,7 @@ namespace xclcpuemhal2 {
     return true;
   }
 
-  bool validateXclBin(const xclBin *header, std::string &xclBinName)
+  bool validateXclBin(const xclBin *header, std::string &xclBinName, bool &newFlow)
```

Comment:
Can you please give some name to newflow.

Reply:
@chvamshi-xilinx , comments addressed and submitted change again, please check.